### PR TITLE
Mockup UI for DKG/Eggnet Statistics Page

### PR DESCRIPTION
**Pull request by AtelyPham**
- feat: add the dkg/eggnet statistics tab
- feat: add statistics overview section
- feat: add dkg signers table
- feat: add chart section
- chore: add label color for statistic card
- feat: add progress bar for uptime value
- chore: update width of child tab on sidebar and add border for header button on dark mode

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [x] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## What is the current behavior?
We don't have the statistic tab on the sidebar and no existing statistics for DKG/Eggnet.

Issue Number: Closes https://github.com/webb-tools/webb-dapp/issues/434

## What is the new behavior?

https://user-images.githubusercontent.com/60747384/175298145-00f9a843-71fa-4b8e-8ba3-be7e084c85f1.mp4

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

## Other information
Data on the page is seeding randomly. Need to implement the API query to get real data.

### CODE REVIEW

File: packages/page-statistics/src/DKGEggnetStatistics/useDKGEggnetStats.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+const initialState = {
+  blockNumber: 0,
+  signatureThreshold: 0,
+  keygenThreshold: 0,
+  authorities: 0,
+  proposerCount: 0,
+  signedProposals: 0,
+  unsignedProposalQueue: 0,
+  dkgSigners: [] as DKGSigner[],
+} as DKGEggnetData;
```
Comment (AhmedKorim) [inline packages/page-statistics/src/DKGEggnetStatistics/useDKGEggnetStats.ts:31]:
```suggestion
const initialState: DKGEggnetData = {
  blockNumber: 0,
  signatureThreshold: 0,
  keygenThreshold: 0,
  authorities: 0,
  proposerCount: 0,
  signedProposals: 0,
  unsignedProposalQueue: 0,
  dkgSigners: [],
};
```

File: packages/page-statistics/src/DKGEggnetStatistics/useDKGEggnetStats.ts [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+export function useDKGEggnetStats() {
+  const [state, setState] = useState<DKGEggnetData>(initialState);
+  const [isLoading, setIsLoading] = useState(false);
+  // const [error, setError] = useState<Error | null | boolean>(null);
```
Comment (AhmedKorim) [inline packages/page-statistics/src/DKGEggnetStatistics/useDKGEggnetStats.ts:36]: Can we remove the comment, if no need for future use?

File: packages/page-statistics/src/index.tsx [hunk 3]
```diff
@@ -2,13 +2,14 @@ import React, { FC } from 'react';
...
 import { RelayersStatistics } from './RelayerStatistics';
 import { StatisticsOverview } from './StatisticsOverview';
 import { WithdrawalStatistics } from './WithdrawalStatistics';
 
 type StatisticsPageProps = {
-  view: string;
+  view: 'overview' | 'deposits' | 'withdrawals' | 'relayers' | 'dkg' | 'dkg-eggnet';
 };
 
 const PageStatistics: FC<StatisticsPageProps> = ({ view }) => {
```
Comment (AhmedKorim) [inline packages/page-statistics/src/index.tsx:15]: Can we use switch here for clarity, and ts would till if there is unhandled branch

**Review by AhmedKorim — CHANGES_REQUESTED** [review]
Great job @AtelyPham 🔥

**Comment by AtelyPham** [comment]
New UI update

https://user-images.githubusercontent.com/60747384/175349606-bf597036-e571-4d5a-bfe1-2f354d4e6a03.mp4

**Review by AhmedKorim — APPROVED** [review]

**Pull request merged**